### ISSUE TITLE
Failed build under Xubuntu 20.04

### ISSUE THREAD

**Issue by Skif-off** [mvo5/synaptic#55]
```sh
# apply distro specific patches if we have them
if [ -e debian/patches/00list.Ubuntu ]; then \
  cp debian/patches/00list.Ubuntu debian/patches/00list;\
  dpatch apply-all;\
fi;
applying patch 01_ubuntu_changelog to ./ ... failed.
```
As I see patch for `common/rpackage.cc` is too old, I tried to delete it and keep patch for `common/rpackagestatus.cc`, but I got the same error.

**Comment by Vistaus** [comment]
Isn't this fixed by the updated patch from #67?

**Comment by Skif-off** [comment]
Maybe [cf4e2711f5c02ad468b62a60256240464e4a5e95](https://github.com/mvo5/synaptic/commit/cf4e2711f5c02ad468b62a60256240464e4a5e95) solved this problem?
I tried to build 0.90.2 and everything went fine, no errors or warnings. So, I will close this issue as solved.
